test/Renci.SshNet.TestTools.OpenSSH/SshdConfig.cs

```diff
@@ -385,6 +385,8 @@ private static void ProcessGlobalOption(SshdConfig sshdConfig, string line)
                 case "AuthorizedKeysFile":
                 case "PasswordAuthentication":
                 case "GatewayPorts":
+                // Had to add this otherwise docker container setup in integration test fails.
+                case "Include":
```

Comment:
No need for the comment

```suggestion
                case "Include":
```

Reply:
OK, removed the comment.